src/resources/postcss/components/_events-bar.pcss

```diff
@@ -7,23 +7,78 @@
 	.tribe-events__events-bar {
 		display: flex;
 		width: 100%;
-		flex-direction: row;
 		flex-wrap: wrap;
-		align-items: center;
+
+		@media (--viewport-medium) {
+			flex-direction: row;
+			align-items: center;
+		}
 	}
 
 	.tribe-events__events-bar-form {
-		flex: 1 1 45%;
+		flex: 1 1 100%;
+		display: none;
+		order: 3;
+
+		@media (--viewport-medium) {
+			display: flex;
+			flex: 1 1 45%;
+			order: 1;
+		}
 	}
 
 	.tribe-events__events-bar-views {
-		flex: 1 1 40px;
+		flex: 1 1 60%;
 		display: flex;
 		justify-content: center;
+		order: 1;
+
+		@media (--viewport-medium) {
+			flex: 1 1 40px;
+			order: 2;
+		}
+
+		.tribe-common-form-control-tabs {
```

Comment:
This is to set the control tabs (select in small viewports) to 100% instead of a fixed width (to make it fluid). Do you think we can cover that on the common styles or should I define a new specification (and new class) here?